package.json

```diff
@@ -6,9 +6,12 @@
   "scripts": {
     "start": "node dist/server/app.js",
     "start:dev": "babel-watch src/server/app.js",
+    "test": "NODE_ENV=test npm run db:migrate && mocha src/server/test/**  --require babel-polyfill --compilers js:babel-register --exit && npm run undo:migrate",
```

Comment:
@mikey2020 @ayodelevm @3m3kalionel what do you feel about us using **jest** for testing both on client side and server-side?

Reply:
Make sense